### DISCUSSION TITLE
Add NFT on-chain as avatars for users

### DISCUSSION THREAD

**Discussion by AmAzing129** [farcasterxyz/protocol#155] [FIP Stage 1: Ideas]
Can we add NFT on-chain as avatars for users?
I don't know how to do it but I think it's a great idea.
On Twitter or other social media, avatars are just pngs though NFTs are popular.
But maybe we can use farcaster to link NFT on-chain to users on-chain?
Anyone can help?

**Comment by ghost** [comment]
> Can we add NFT on-chain as avatars for users?
> 
> I don't know how to do it but I think it's a great idea.
> 
> On Twitter or other social media, avatars are just pngs though NFTs are popular.
> 
> But maybe we can use farcaster to link NFT on-chain to users on-chain?
> 
> Anyone can help?

bro it's already like this, just go to an NFT you like and press "use as pfp"
